src/sage/categories/finite_dimensional_lie_algebras_with_basis.py

```diff
@@ -70,27 +70,27 @@ def _construct_UEA(self):
 
             EXAMPLES::
 
-                sage: L = LieAlgebras(QQ).FiniteDimensional().WithBasis().example()
-                sage: UEA = L._construct_UEA(); UEA
+                sage: L = LieAlgebras(QQ).FiniteDimensional().WithBasis().example()                                     # optional - sage.modules sage.combinat
+                sage: UEA = L._construct_UEA(); UEA                                                                     # optional - sage.modules sage.combinat
                 Noncommutative Multivariate Polynomial Ring in b0, b1, b2
                  over Rational Field, nc-relations: {}
-                sage: UEA.relations(add_commutative=True)
+                sage: UEA.relations(add_commutative=True)                                                               # optional - sage.modules sage.combinat
                 {b1*b0: b0*b1, b2*b0: b0*b2, b2*b1: b1*b2}
 
             ::
 
-                sage: L.<x,y,z> = LieAlgebra(QQ, {('x','y'):{'z':1}, ('y','z'):{'x':1}, ('z','x'):{'y':1}})
-                sage: UEA = L._construct_UEA(); UEA
+                sage: L.<x,y,z> = LieAlgebra(QQ, {('x','y'):{'z':1}, ('y','z'):{'x':1}, ('z','x'):{'y':1}})             # optional - sage.modules sage.combinat
```

Comment:
Could split this line and align all the labels to column 88.

Reply:
Good idea, done